x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -179,9 +179,14 @@ public class EsqlFeatures implements FeatureSpecification {
      */
     public static final NodeFeature RESOLVE_FIELDS_API = new NodeFeature("esql.resolve_fields_api");
 
+    /**
+     * Support match operator syntax
+     */
+    public static final NodeFeature MATCH_SYNTAX = new NodeFeature("esql.match_syntax");
```

Comment:
We've moved away from adding these. I put a capability in the gist I just commented that doesn't the same kind of thing.

Reply:
The rest of the gist is useless though.

Reply:
It's using capabilities now - thanks for the gist!